src/condor_utils/param_info.in

```diff
@@ -6793,6 +6793,19 @@ default : @fin
 			SET Requirements $(addl_reqs)
 		endif
 	@end
+
+	if $(ANNEX_JOBS_EXCLUSIVELY:TRUE)
+		JOB_TRANSFORM_NAMES = $(JOB_TRANSFORM_NAMES) AnnexJobsExclusively
+		JOB_TRANSFORM_AnnexJobsExclusively @=end
+			REQUIREMENTS TargetAnnexName =?= undefined
+			addl_reqs = ifThenElse(TARGET.AuthenticatedIdentity is undefined, true, !regexp(".*@$(ANNEX_TOKEN_DOMAIN)", TARGET.AuthenticatedIdentity))
```

Comment:
```suggestion
			addl_reqs = ifThenElse(TARGET.AuthenticatedIdentity is undefined, true, !regexp(".*@$(ANNEX_TOKEN_DOMAIN)$", TARGET.AuthenticatedIdentity))
```
In case you haven't gotten to this yet.